plugin/helpers.go

```diff
@@ -29,8 +33,18 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.6
 	KVSetWithExpiryJSON(key string, value interface{}, expireInSeconds int64) error
+
+	// Register a command, using the Plugin API, and stores the callback attached to the command trigger.
```

Comment:
Propose tweaking to godoc conventions. I don't it's necessary to call out the error conditions:
```
	// RegisterCommand registers the given command and callback for the ExecuteCommand helper.
```